src/libs/runtime-utils.js

```diff
@@ -265,7 +286,8 @@ export const getCurrentApp = appType => getCurrentAppExcludingStatuses(appType,
 export const getCurrentAppIncludingDeleting = appType => getCurrentAppExcludingStatuses(appType, [])
 
 export const getCurrentAttachedDataDisk = (app, appDataDisks) => {
-  return _.find({ name: app?.diskName }, appDataDisks)
+  const disk = _.find({ name: app?.diskName }, appDataDisks)
+  return disk && _.set('diskType', pdTypes.fromString(disk.diskType), disk)
```

Comment:
Sorry that I missed this earlier. Can we be explicit about type coercion in cases like this? 
 
```suggestion
  return !!disk && _.set('diskType', pdTypes.fromString(disk.diskType), disk)
```